From 6b6d7594cdfd18f9b13a4e8951c039d54f8c4a6d Mon Sep 17 00:00:00 2001
From: Suresh Gupta <suresh.gupta@freescale.com>
Date: Fri, 26 Apr 2013 13:02:38 +0530
Subject: [PATCH 069/124] fsl/usb: Add NULL entry in platform_device_id

Gadget compilation breaks without terminated NULL entry in platform_device_id

Signed-off-by: Suresh Gupta <suresh.gupta@freescale.com>
Change-Id: I5821f0bf73a3cdfdb7988079ac97237e9a3f5f8b
Reviewed-on: http://git.am.freescale.net:8181/2124
Reviewed-by: Li Yang-R58472 <LeoLi@freescale.com>
Reviewed-by: Mehresh Ramneek-B31383 <ramneek.mehresh@freescale.com>
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
---
 drivers/usb/gadget/udc/fsl_udc_core.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/usb/gadget/udc/fsl_udc_core.c b/drivers/usb/gadget/udc/fsl_udc_core.c
index 445dcf4..d9cfe6b 100644
--- a/drivers/usb/gadget/udc/fsl_udc_core.c
+++ b/drivers/usb/gadget/udc/fsl_udc_core.c
@@ -2677,7 +2677,9 @@ static const struct platform_device_id fsl_udc_devtype[] = {
 	}, {
 		.name = "imx-udc-mx51",
 	}, {
-		.name = "fsl-usb2-udc"
+		.name = "fsl-usb2-udc",
+	}, {
+		/* sentinel */
 	}
 };
 MODULE_DEVICE_TABLE(platform, fsl_udc_devtype);
-- 
2.1.0.27.g96db324

